lib/src/blaze/trial_decryptions.rs

```diff
@@ -289,10 +289,11 @@ impl TrialDecryptions {
                 let config = config.clone();
 
                 workers.push(tokio::spawn(async move {
-                    let keys = keys.read().await;
-                    let wallet_key = &D::Key::get_keys(&*keys)[ivk_num];
-                    let fvk = wallet_key.fvk().unwrap();
-                    let have_spending_key = wallet_key.spend_key().is_some();
+                    let usa = usa.read().await;
+                    let fvk = D::Key::usc_to_fvk(&*usa);
+
+                    // We don't have fvk import, all our keys are spending
```

Comment:
How about we have an issue that tracks the state of viewing key imports?   We're not certain that it's worth supporting.